client/app/components/routes/select-operator/Page.tsx

```diff
@@ -5,10 +5,28 @@ import { BC_GOV_LINKS_COLOR } from "@/app/styles/colors";
 import Link from "next/link";
 import { getServerSession } from "next-auth";
 import { authOptions } from "@/app/api/auth/[...nextauth]/route";
+import { actionHandler } from "@/app/utils/actions";
+import { Status } from "@/app/types/types";
 
-export default async function SelectOperatorPage() {
+const getUserOperatorStatus = async () => {
+  try {
+    return await actionHandler(
+      `registration/user-operator-status-from-user`,
+      "GET",
+      "",
+    );
+  } catch (error) {
+    throw error;
+  }
+};
+
+export default async function MyOperatorPage() {
```

Comment:
I made a tech debt card to consider renaming the route now that it does more than select operator: https://app.zenhub.com/workspaces/team-moose-sprint-board-60d216175a2689000ec774d0/issues/gh/bcgov/cas-registration/444 (I expect the approved/rejected workflows will go through this page too)